src/components/Auth/ResetPassword.vue

```diff
@@ -0,0 +1,118 @@
+<template>
+  <v-container align-center style="background: transparent;" fluid class="resetPasswordForm">
+    <v-flex xs12 md6 class='reset-password-flex mx-auto'>
+      <v-row>
+        <v-flex align-center xs12 fluid text-center>
+          <h2>Reset Password</h2>
+        </v-flex>
+      </v-row>
+      <v-row>
+        <v-flex align-center xs12 fluid text-center>
+          <h5>Enter your new password, and reenter to confirm. Minimum 6 characters.</h5>
+        </v-flex>
+      </v-row>
+      <v-row class='pa-md-4 white--text' v-if='checkErrors'>
+        Your passwords do not match.
+      </v-row>
+      <v-row>
+        <v-flex xs12 fluid text-center>
+          <v-text-field
+            v-model="password"
+            outlined
+            label="password"
+            name="Password"
+            color="#B9BCC1"
+            type="password"
+            required
+            dark
+          >
+          <v-icon slot="append" color="#B9BCC1">lock</v-icon>
+          </v-text-field>
+        </v-flex>
+      </v-row>
+      <v-row>
+        <v-flex xs12 fluid text-center>
+          <v-text-field
+            v-model="confirmPassword"
+            outlined
+            label="Confirm Password"
+            name="ConfirmPassword"
+            color="#B9BCC1"
```

Comment:
Instead of hardcoded color codes, we should use named colors. named colors can be found in themes.ts file.